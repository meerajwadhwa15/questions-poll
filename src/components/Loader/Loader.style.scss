.loader {
    color: $brand-black;
    text-align: center;
    margin: 0 auto;
}