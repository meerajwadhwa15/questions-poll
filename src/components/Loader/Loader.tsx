import React, { FunctionComponent } from "react";

export type Props = {};

const Loader: FunctionComponent<Props> = () => <div className="loader">Loading...</div>;

export default Loader;
