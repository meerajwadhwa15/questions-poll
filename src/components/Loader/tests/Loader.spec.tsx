import React from "react";
import Loader from "../Loader";
import { create } from "react-test-renderer";

const props = {
};

test("Matches the snapshot with Loader", () => {
  const component = create(
      <Loader {...props} />
  );
  expect(component.toJSON()).toMatchSnapshot();
});
