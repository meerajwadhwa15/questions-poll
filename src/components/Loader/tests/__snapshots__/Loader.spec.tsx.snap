// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot with Loader 1`] = `
<div
  className="loader"
>
  Loading...
</div>
`;
